src/components/ChatLog.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+    const chatComp = props.entries.map((messages) => {
+        return (
+            <ChatEntry
+                key={messages.id}
+                id={messages.id}
+                sender={messages.sender}
+                body={messages.body}
+                timeStamp={messages.timeStamp}
+                liked={messages.liked}
+                onUpdateChat={props.onUpdateChat}
+            />
+        );
+    });
+
+    return (
+        <div>
+        {chatComp}
+        </div>
+    );
+};
+
+ChatLog.propTypes = {
+    messages: PropTypes.arrayOf(PropTypes.shape({
+        id: PropTypes.number.isRequired,
+        sender: PropTypes.string.isRequired,
+        body: PropTypes.string.isRequired,
+        timeStamp: PropTypes.string.isRequired,
+        liked: PropTypes.bool.isRequired,
+})),
+onUpdateChat: PropTypes.func
+};
```

Comment:
Nice job using `arrayOf` and `shape`! Note that you passed the chat data down as a prop called `entries` though, which means the message data currently is not covered by prop types. Make sure your prop type names match the prop names.

Also, I'd recommend making the chat entries array required and the `onUpdateChat` function required since both of these are essential for the app to work properly.

Ex:
```
ChatLog.propTypes = {
    entries: PropTypes.arrayOf(PropTypes.shape({
        id: PropTypes.number.isRequired,
        sender: PropTypes.string.isRequired,
        body: PropTypes.string.isRequired,
        timeStamp: PropTypes.string.isRequired,
        liked: PropTypes.bool.isRequired
    })).isRequired,
    onUpdateChat: PropTypes.func.isRequired
};
```